packages/core/package.json

```diff
@@ -31,7 +31,7 @@
     "@cosmjs/tendermint-rpc": "0.29.5",
     "@cosmjs/utils": "0.29.5",
     "@keplr-wallet/types": "^0.11.41",
-    "@sei-js/proto": "1.1.6",
+    "@sei-js/proto": "^0.0.0-internal-20230301223319",
```

Comment:
can you undo these version changes - i think you committed the updates from deploying the `internal` versions of the packages